comid/rawvalue.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type RawValue struct {
```

Comment:
Perhaps we should mention what this RawValue is for and its relevance in CoMID?

Reply:
sure, done in e1e9225